README.md

```diff
@@ -41,37 +41,101 @@ go get github.com/bonitoo-io/influxdb3-go
 
 ## Usage
 
-To start with the client, import the `influx` package and create a `influx.Client` by the `NewClient` function:
+set environment variables:
+
+- `INFLUXDB_URL` region of your influxdb cloud e.g *`https://us-east-1-1.aws.cloud2.influxdata.com/`*
+- `INFLUXDB_TOKEN` read/write token generaterd in cloud e.g *`my-bucket`*
+- `INFLUXDB_BUCKET` name of bucket
+
+<details>
+  <summary>linux/macos</summary>
+
+```sh
+export INFLUXDB_URL="<url>"
+export INFLUXDB_BUCKET="<bucket>"
+export INFLUXDB_TOKEN="<token>"
+```
+
+</details>
+
+<details>
+  <summary>windows</summary>
+
+```powershell
+setx INFLUXDB_URL "<url>"
+setx INFLUXDB_BUCKET "<bucket>"
+setx INFLUXDB_TOKEN "<token>"
+```
+
+</details>
+
+To get started with influxdb client import `influxdb3-go` package.
 
 ```go
 import (
-    "github.com/bonitoo-io/influxdb3-go/influx"
-    "github.com/bonitoo-io/influxdb3-go/influx/configs"
+  "context"
+  "encoding/json"
+  "fmt"
+  "os"
+
+  "github.com/bonitoo-io/influxdb3-go/influx"
 )
+```
 
-host := "https://eu-central-1-1.aws.cloud2.influxdata.com/"
-database := "my-database"
-token := "my-token"
+Create `influx.Client` with `New` function. Make sure to `Close` client after with `defer` keyword.
 
-client, err := NewClient(configs.ClientConfigs{
-    Host:     &host,
-    Database: &database,
-    Token:    &token},
-)
+```go
+url := os.Getenv("INFLUXDB_URL")
+token := os.Getenv("INFLUXDB_TOKEN")
+bucket := os.Getenv("INFLUXDB_BUCKET")
+
+// Create a new client using an InfluxDB server base URL and an authentication token
+client, err := influx.New(influx.Params{
+    ServerURL: url,
+    AuthToken: token,
+})
+defer client.Close()
 ```
 
-to insert data, you can use code like this:
+The `client` can be now used to insert data using [line-protocol](https://docs.influxdata.com/influxdb/cloud-serverless/reference/syntax/line-protocol/)
 
 ```go
-// TBD
+line := fmt.Sprintf("stat,unit=temperature avg=%f,max=%f", 23.5, 45.0)
+err = client.Write(context.Background(), bucket, []byte(line))
 ```
 
-to query your data, you can use code like this:
+Fetch data using FlightSQL query and print result.
 
 ```go
-// TBD
+query := `
+        SELECT *
+        FROM "stat"
+        WHERE
+        time >= now() - interval '5 minute'
+        AND
+        "unit" IN ('temperature')
+`;
+
+reader, err := client.Query(context.Background(), bucket, query, nil)
+
+for reader.Next() {
+    record := reader.Record()
+    b, err := json.MarshalIndent(record, "", "  ")
+    if err != nil {
+        panic(err)
+    }
+    fmt.Println(string(b))
+
+    if err := reader.Err(); err != nil {
+        panic(err)
```

Comment:
Please add code to show how to operate with `Arrow.Record`... how to iterate over database rows.

Reply:
This library should help users to work with FlightSQL and InfluxDB IOx. Try to look how it is implemented in C#: 

- https://github.com/bonitoo-io/influxdb3-csharp/blob/614b22ae6e1425d5273cd3527b72bec9ed6226ac/Client/Internal/FlightSqlExtensions.cs#L18
- https://github.com/bonitoo-io/influxdb3-csharp/blob/614b22ae6e1425d5273cd3527b72bec9ed6226ac/Client/InfluxDBClient.cs#L141

Something like this can work:

```golang
for reader.Next() {
		record := reader.Record()
		rowCount := record.Column(0).Len()
		for i := 0; i < rowCount; i++ {
			for j := 0; j < len(record.Columns()); j++ {
				array := record.Column(j)
				value := array.ValueStr(i)
				switch v := array.DataType().(type) {
				case *arrow.BooleanType:
					boolValue, err := strconv.ParseBool(value)
					if err != nil {
						log.Fatal(err)
					}
					// append to results as Boolean
				case ...
				default:
					fmt.Printf("I don't know about type %T!\n", v)
				}
			}
		}
		b, err := json.MarshalIndent(record, "", "  ")
		if err != nil {
			panic(err)
		}
		fmt.Println(string(b))

		if err := reader.Err(); err != nil {
			panic(err)
		}
	}
```